### DISCUSSION TITLE
What is the bootstrap round ('num_rounds' param in mlxtend.evaluate bias_variance_decomp)?

### DISCUSSION THREAD

**Discussion by diman82** [rasbt/mlxtend#786] [Q&A] [answered]

**Comment by rasbt** [comment, marked as answer]
Hi there. It's the number of bootstrap rounds. I.e., how  many bootstrap samples are being drawn. Let me know if you need additional clarification.

**Reply by diman82** [reply]
Hi.
Bootstrap samples are being drawn from where? From viewing the source code it seems, that the function randomly re-samples data (from the train set) and runs num_rounds of iterations (.fit()) with the newly sampled data.

**Reply by rasbt** [reply]
Yes, that correct. Thanks for bringing this up, this should be described in the docstring; will fix that!
